pmm-tests/pmm-framework.sh

```diff
@@ -857,6 +857,19 @@ remove_packages(){
   fi
 }
 
+install_pgsql_plugin(){
+  yum install -y gcc readline-devel zlib-devel
+  wget https://ftp.postgresql.org/pub/source/v10.6/postgresql-10.6.tar.gz
+  tar -xf postgresql-10.6.tar.gz
```

Comment:
@puneet0191 only 10.6 version?

Reply:
@Nailya  this function is removed, please refresh..it is outdated